cmd/up/activate.go

```diff
@@ -293,7 +293,7 @@ func (up *upContext) waitUntilDevelopmentContainerIsRunning(ctx context.Context)
 				}
 				continue
 			}
-			log.Infof("pod %s event: %s", up.Pod, e.Message)
```

Comment:
@jLopezbarb @pchico83 I'm removing this log because it logs the entire pod object a few times. If you feel is important let me know, and maybe we can change it to just the name of the pod, or some key values?